src/site/content/en/blog/optimize-cls/index.md

```diff
@@ -41,6 +41,7 @@ The most common causes of a poor CLS are:
 
   <figure class="w-figure">
     <video controls loop muted class="w-screenshot" poster="video-image-dimensions.jpg">
+      <source src="https://storage.googleapis.com/web-dev-assets/optimize-cls/image-dimensions.webm" type="video/webm;">
```

Comment:
WebM videos here won't load until they've been uploaded to the Cloud bucket. Pinged Kayce to request :)

Reply:
Done